snippet-extractor-output/snippets.yaml

```
@@ -1258,6 +1258,54 @@
 
         await context.sync();        
     });
+'Excel.ChartLeaderLines:class':
+  - >-
+    // Link to full sample:
+    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/excel/10-chart/chart-leader-lines.yaml
+
+
+    // This function changes the format of leader lines. 
+
+    // It adjusts color, weight, and line style.
+
+    await Excel.run(async (context) => {
+      const sheet = context.workbook.worksheets.getItem(sheetName);
+      const chart = sheet.charts.getItemAt(0);
+      const series = chart.series.getItemAt(0);
+      const dataLabels = series.dataLabels;
+      const lineFormat = dataLabels.leaderLines.format;
+
+      // Set leader line formatting properties.
+      lineFormat.line.color = "blue";
+      lineFormat.line.weight = 2;
+      lineFormat.line.lineStyle = Excel.ChartLineStyle.dot;
+
+      await context.sync();
+    });
+'Excel.ChartLeaderLinesFormat:class':
+  - >-
+    // Link to full sample:
+    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/excel/10-chart/chart-leader-lines.yaml
+
+
+    // This function changes the format of leader lines. 
+
+    // It adjusts color, weight, and line style.
```

Maybe combine this with previous comment.

```suggestion
    // The following code adjusts color, weight, and line style.
```